context/package.json

```diff
@@ -109,7 +110,7 @@
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.11.0",
     "webpack-manifest-plugin": "^2.2.0",
-    "webpack-merge": "^4.2.2",
+    "webpack-merge": "^5.8.0",
```

Comment:
This is what prompts these changes? 
```diff
- const merge = require('webpack-merge');
+ const { merge } = require('webpack-merge');
```

Reply:
I needed to use `mergeWithCustomize` and `unique` seen in `webpack.analyze.js` to specifically override the `clean-webpack-plugin`. By default `merge` just concats the plugins array.